ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -552,7 +534,7 @@ - (void)tryRefreshToken:(ADAuthenticationCallback)completionBlock
     ADAcquireTokenSilentHandler *request = [ADAcquireTokenSilentHandler requestWithParams:_requestParams
                                                                                tokenCache:self.tokenCache];
     [request acquireTokenByRefreshToken:_refreshToken
-                              cacheItem:[ADTokenCacheItem new]
+                              cacheItem:nil
```

Comment:
I didn't find other places where we pass `cacheItem`. Can we remove that parameter?